packages/components/src/font-size-picker/stories/e2e/index.tsx

```diff
@@ -13,8 +18,11 @@ export default {
 	component: FontSizePicker,
 };
 
-const FontSizePickerWithState = ( { initialValue, ...props } ) => {
-	const [ fontSize, setFontSize ] = useState( initialValue );
+const FontSizePickerWithState: ComponentStory< typeof FontSizePicker > = ( {
+	value,
```

Comment:
Tiniest of nits: we could rename `value` to `initialValue` to make its purpose more explicit

```suggestion
	value: initialValue,
```

Same would apply for the Storybook story

Reply:
`value` is the name of the prop that `FontSizePicker` accepts though.

Reply:
Gotcha — I'm also fine keeping it as is